src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -21,15 +22,26 @@
 (defn get-keypair-data
   [name derivation-path]
   [{:title             (keypair-string name)
-    :button-props      {:title (i18n/label :t/edit)}
-    :left-icon         :i/placeholder
+    :image             :avatar
+    :image-props       {:full-name           "A Y"
```

Comment:
the full name is the users account name which we have access to? 
also the customization color is the users custom color which we also have/ or it should be the new account color which I believe we also have 🤔